types/rest/rest.go

```diff
@@ -199,9 +199,9 @@ func PostProcessResponse(w http.ResponseWriter, cliCtx client.Context, resp inte
 		return
 	}
 
-	switch resp.(type) {
+	switch resp := resp.(type) {
 	case []byte:
-		result = resp.([]byte)
+		result = resp
 
```

Comment:
@jdkanani @anilCSE please check if this is ok. Ref: https://nanxiao.gitbooks.io/golang-101-hacks/content/posts/type-assertion-and-type-switch.html